start-database.sh

```diff
@@ -29,7 +29,7 @@ fi
 
 # import env variables from .env
 set -a
-source .env
+source ./.env
```

Comment:
is the change from `source .env` to `source ./.env` necessary? Since `start-database.sh` and `.env` are in the same directory, specifying ./ might be redundant. Could you check if the script works without this change?"

Reply:
It does, i just fixed that.